iothub/device/src/IotHubBaseClient.cs

```diff
@@ -208,6 +218,11 @@ public async Task SendTelemetryAsync(TelemetryMessage message, CancellationToken
         /// <exception cref="OperationCanceledException">The operation has been canceled.</exception>
         /// <exception cref="IotHubClientException">An error occured when communicating with IoT hub service.</exception>
         /// <exception cref="ObjectDisposedException">The client has been disposed.</exception>
+        /// <example>
+        /// <code language="csharp">
+        /// await client.SendTelemetryAsync(new List<TelemetryMessage/> { message1, message2 }, cancellationToken);
```

Comment:
should it be like:

/// await client.SendTelemetryAsync(new List &lt;TelemetryMessage&gt; { message1, message2 }, cancellationToken);

Reply:
check my comment above